packages/server/routes/api/auth.js

```diff
@@ -2,11 +2,18 @@ const express = require("express");
 const passport = require("passport");
 const router = express.Router();
 const token = require("../../utils/token");
-const { OAuth2Client } = require('google-auth-library');
+const { OAuth2Client, UserRefreshClient } = require('google-auth-library');
 
 const GoogleStrategy = require("passport-google-oauth20").Strategy;
 
 const AuthController = require("../../controllers/auth");
+const { response } = require("express");
```

Comment:
This might be the result of VSCode autocomplete. You should remove it since it's an unnecessary import.